### ISSUE TITLE
ASPIRE FFT and NUFFT Tutorial Example

### ISSUE THREAD

**Issue by garrettwrong** [ComputationalCryoEM/ASPIRE-Python#977]
User asked how to use our FFT and NUFFT components.  These components are documented, but we don't have a short getting started tutorial.  Should be easy enough to make one that shows:

- FFT and Inverse on 1,2,3 dimensions.
- FFT with stack axis
- Configuring a different backend in config
- NUFFT to resample onto a grid
- (Configuring a different backend in config, noting we currently only have one option here #276 ).

Good rainy day project.

**Comment by janden** [comment]
Right. Since this is an internal component, I wouldn't put it too high up the priority list, however.
